### PR TITLE
Add Editor Integration section to README

### PR DESCRIPTION
**Pull request by rgreenjr**
## Description

This pull request adds an **Editor Integration** section to the README. It describes the new Atom `exfmt-atom` plugin and will hopefully include more integrations soon.

## Checklist

- [x] The change has been discussed in a GitHub issue.
- [] There are tests for the new functionality.
- [x] I agree to adhere to the code of conduct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ The benefits of `exfmt`:
 ## Contents
 
 - [Usage](#usage)
+- [Editor Integration](#editorintegration)
```
Comment (jfornoff) [inline README.md]: I think you need a dash in the anchor (`editor-integration`) to work with the heading below.
You can check it working in the `Files changed` tab and hitting the `View` button on the top right! ;-)

**Review by jfornoff — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rgreenjr) [reply README.md]: Thanks for the heads-up.

File: README.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ The benefits of `exfmt`:
 ## Contents
 
 - [Usage](#usage)
+- [Editor Integration](#editor-integration)
```
Comment (lpil) [inline README.md]: Could you add a sub-item here please? :)

```
- [Editor Integration](#editor-integration)
  - [Atom](#atom)
```

Comment (rgreenjr) [reply README.md]: Absolutely.

**Comment by lpil** [comment]
Thank you @rgreenjr ! :D

**Pull request merged**